### PR TITLE
Oceanwater 484 add function input description

### PR DESCRIPTION
**Pull request by damianacatanoso**
Ticket: https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-484

To each function definition in the ow_lander>scripts files, I added description of the input type. 

Please take a look at the format and tell me if it looks good. Thanks!

### CODE REVIEW

File: ow_lander/scripts/activity_guarded_move.py [hunk 1]
```diff
@@ -89,10 +92,18 @@ def pre_guarded_move(move_arm, args):
...
   return True
 
-def guarded_move(move_arm, args):
+def guarded_move(move_arm,     # type: class 'moveit_commander.move_group.MoveGroupCommander' 
+                 args):        # type: List[bool, bool, float, float, float, float, float, float, float] 
+
+  print "guarded_move"
+  print type(args)
+  for i in args:
+    print type(i)
```
Comment (Samahu) [inline ow_lander/scripts/activity_guarded_move.py:106]: These seem for debugging
```suggestion
```

File: ow_lander/scripts/ground_detection.py [hunk 2]
```diff
@@ -108,10 +124,12 @@ def _tf_lookup_position(self, timeout=rospy.Duration(0.0)):
     return t.transform.translation
 
   def _tf_2_method(self):
+
     return self._check_condition(self._tf_lookup_position())
 
   @property
   def ground_position(self):
+
```
Comment (Samahu) [inline ow_lander/scripts/ground_detection.py:132]: Why these new lines being added?
```suggestion

```

File: ow_lander/scripts/ground_detection.py [hunk 3]
```diff
@@ -120,6 +138,7 @@ def ground_position(self):
     return Point(position[0], position[1], position[2])
 
   def detect(self):
+
```
Comment (Samahu) [inline ow_lander/scripts/ground_detection.py:141]: Same here
```suggestion
```

File: ow_lander/scripts/mechanical_power_arm.py [hunk 4]
```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env python2
-
```
Comment (Samahu) [inline ow_lander/scripts/mechanical_power_arm.py:2]: I support leaving an empty line between the environment and the imports

File: ow_lander/scripts/utils.py [hunk 5]
```diff
@@ -32,7 +38,13 @@ def start_traj_recording(delete_prev_traj,bagname):
...
   p = subprocess.Popen(command, stdin=subprocess.PIPE, shell=True, cwd='.')
 
-def stop_traj_recording(result, bagname):
+def stop_traj_recording(result,       # type: bool
+                        bagname):     # type: str
+
+  print "utils>stop_traj_recording"
+  print type(result)
+  print type(bagname)
+
```
Comment (Samahu) [inline ow_lander/scripts/utils.py:47]: Are all of these for debugging the types
```suggestion
```

File: ow_lander/scripts/utils.py [hunk 6]
```diff
@@ -56,11 +68,19 @@ def stop_traj_recording(result, bagname):
   command = "rm " + bagname + ".bag"
   os.system(command)
 
-def is_shou_yaw_goal_in_range(joint_goal):
+def is_shou_yaw_goal_in_range(joint_goal):      # type: List[float, float, float, float, float, float]
+  
+  print "utils>is_shou_yaw_goal_in_range"
+  print type(joint_goal)
+  for i in joint_goal:
+    print type(i)
```
Comment (Samahu) [inline ow_lander/scripts/utils.py:76]: Debug prints
```suggestion
```

**Review by Samahu — CHANGES_REQUESTED** [review]
I request that you remove the debug prints.

Also, rather than using this non-standard way to add info about a method parameters I would recommend using docstring which works for python 2 & python.

**Comment by Samahu** [comment]
You can see examples of the use docstring in some of the modules that I have added

File: ow_lander/scripts/activity_guarded_move.py (hunk 1 above)
Comment (damianacatanoso) [reply ow_lander/scripts/activity_guarded_move.py:106]: Oh yes, sure, I'll remove those. I thought I removed them all.

File: ow_lander/scripts/ground_detection.py (hunk 2 above)
Comment (damianacatanoso) [reply ow_lander/scripts/ground_detection.py:132]: Some methods had empty lines after the definition lines, some others didn't. So I added an empty line to the ones that didn't have it for consistency. I'll remove them.

File: ow_lander/scripts/mechanical_power_arm.py (hunk 4 above)
Comment (damianacatanoso) [reply ow_lander/scripts/mechanical_power_arm.py:2]: OK!

File: ow_lander/scripts/utils.py (hunk 5 above)
Comment (damianacatanoso) [reply ow_lander/scripts/utils.py:47]: Yes, before writing the type I printed out the type to double check. I'll remove those

**Review by Samahu — APPROVED** [review]

**Review by mogumbo — APPROVED** [review]

**Pull request closed without merging**